.idea/workspace.xml

```diff
@@ -2,16 +2,6 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="3e5fcfae-c61c-4f90-be45-c5ef41070ce5" name="Default Changelist" comment="">
-      <change afterPath="$PROJECT_DIR$/.idea/libraries/Maven__junit_junit_4_12.xml" afterDir="false" />
-      <change afterPath="$PROJECT_DIR$/.idea/libraries/Maven__org_hamcrest_hamcrest_core_1_3.xml" afterDir="false" />
-      <change afterPath="$PROJECT_DIR$/davidengel-app.iml" afterDir="false" />
-      <change afterPath="$PROJECT_DIR$/src/main/java/edu/davidengel/advancedjava/Computer.java" afterDir="false" />
```

Comment:
do not include any compiler, ide, or maven generated files. 
use .gitingore 
I review this in the weekly chat